src/lxc/confile.c

```diff
@@ -4898,6 +5031,24 @@ static int clr_config_net_ipv4_address(const char *key,
 	return 0;
 }
 
+static int clr_config_net_ipv4_route(const char *key,
+				       struct lxc_conf *lxc_conf, void *data)
+{
+	struct lxc_netdev *netdev = data;
+	struct lxc_list *cur, *next;
+
+	if (!netdev)
+		return -1;
+
+	lxc_list_for_each_safe(cur, &netdev->ipv4_routes, next) {
+		lxc_list_del(cur);
+		free(cur->elem);
```

Comment:
@brauner does `__do_free` deal with all properties of variable, i.e if I made 

```
__do_free struct lxc_list *cur;
```

Would that suffice for the ```free(cur->elem);``` line too?

Reply:
no

Reply:
no

Reply:
ok thanks @brauner, so as this is an explicit "clean up" function, are you happy for me to leave the use of `free()` here and not use the `__do_free` macros?

Reply:
Leave it as is.